usage/test-support/src/main/java/org/apache/brooklyn/test/performance/PerformanceMeasurer.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.test.performance;
+
+import static org.testng.Assert.fail;
+
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.brooklyn.util.exceptions.Exceptions;
+import org.apache.brooklyn.util.time.Duration;
+import org.apache.brooklyn.util.time.Time;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.annotations.Beta;
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.Lists;
+
+/**
+ * For running simplistic performance tests, to measure the number of operations per second.
+ * 
+ * With a short run, this is "good enough" for eye-balling performance, to spot if it goes 
+ * horrendously wrong. 
+ * 
+ * However, good performance measurement involves much more warm up (e.g. to ensure java HotSpot 
+ * optimisation have been applied), and running the test for a reasonable length of time.
+ * 
+ * Longevity tests are also important for to check if object creation is going to kill
+ * performance in the long-term, etc.
+ */
+@Beta
+public class PerformanceMeasurer {
```

Comment:
Agree with your other renames.

However `TestRunner` is a technical term in testng (http://testng.org/javadoc/org/testng/TestRunner.html) so that doesn't seem like the right name. I'll stick with the clumsy-but-accurate `PerformanceMeasurer` for now, marked as `@Beta`.